lua/tardis/interiors/templates/templates_default_interior.lua

```diff
@@ -244,6 +217,14 @@ TARDIS:AddInteriorTemplate("default_small_version", {
     },
 })
 
+TARDIS:AddInteriorTemplate("default_small_version_lamp_fix", {
+    Interior = {
+        Size = {
+            Max = Vector(484.983, 514.944, 800)
```

Comment:
What is this for?

Reply:
`default_small_version` is a template that makes the size smaller
`default_lamps` increases the interior height to fit the lamp (we only have one lamp now)

when they are both used, the lamp doesn't fit the small interior size, but this template adds the fix